lib/push.js

```diff
@@ -12,9 +12,12 @@ var LOG_OP_PUSH_TO_DEVICES = 'push.pushToDevices'
 
 var PUSH_PAYLOAD_SCHEMA_VERSION = 1
 var PUSH_COMMANDS = {
-  DEVICE_CONNECTED: 'fxaccounts:device_connected'
+  DEVICE_CONNECTED: 'fxaccounts:device_connected',
+  DEVICE_DISCONNECTED: 'fxaccounts:device_disconnected'
 }
 
+var TTL_DEVICE_DISCONNECTED = 5 * 3600 // 5 hours
```

Comment:
I wonder if it makes sense to derive this value from the lifetime of certificates issued by `/v1/certificate/sign`, since we know that browsers disconnected for longer than this will have to checkin to get a fresh cert and discover the state change that way.